includes/wikia/GlobalTitle.php

```diff
@@ -100,8 +100,14 @@ public static function newMainPage( $city_id = null ) {
 			throw new \Exception( 'Invalid $city_id.' );
 		}
 
-		// sure hope this redirects for the most part
-		$title = self::newFromText( 'Main Page', NS_MAIN, $city_id );
+		$mainPageName = self::newFromText( 'Mainpage', NS_MEDIAWIKI, $city_id )->getContent();
+		$title = self::newFromText( $mainPageName, NS_MAIN, $city_id );
```

Comment:
Just a small question, should main page being outside of main namespace ([example](http://kocka.wikia.com/wiki/MediaWiki:Mainpage)) be handled here?